ext/sockets/tests/socket_getaddrinfo_error.phpt

```diff
@@ -84,6 +84,16 @@ try {
 } catch (\ValueError $e) {
 	echo $e->getMessage() . PHP_EOL;
 }
+try {
+	socket_addrinfo_lookup('127.0.0.1', 2000, [
+		AF_INET,
+		SOCK_DGRAM,
+		0,
+		0,
+	]);
+} catch (\ValueError $e) {
+	echo $e->getMessage() . PHP_EOL;
+}
```

Comment:
Add a test where the array has string keys and integer keys, I'm not exactly sure you are handling that case correctly.

Reply:
another test is already doing it but I can add it sure